src/screens/application/ApplicationStatus.tsx

```
@@ -1,50 +1,87 @@
+    setError(null);
+    try {
+      const result = await getUser();
+
+      const user_id = result?.data?.user_id;
 
+      const data = await getApplicationList(SearchText, user_id);
+
+      setApplicationList(data.data.applications);
+    } catch (error) {
+      throw new Error(`Failed to fetch applications: ${error.message}`);
+      setError("Failed to fetch applications");
```

Biome

[error] This code is unreachable

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)